### PR TITLE
Animated hotel

### PR DESCRIPTION
**Pull request by Ekaterina1994**

### CODE REVIEW

**Review by SergioAJS — APPROVED** [review]

File: gf-webapp/src/logic/bookingPage/form/Form.tsx [hunk 1]
```diff
@@ -4,7 +4,11 @@ import styles from "./Form.module.scss";
 import {useModalVisibilityContext} from "../../../context/Context";
 import {useTranslation} from "react-i18next";
 
-export const Form = () => {
+type Roo = {
```
Comment (tritonJS826) [inline gf-webapp/src/logic/bookingPage/form/Form.tsx]: Roo?) rename please)

File: gf-webapp/src/logic/bookingPage/form/Form.tsx [hunk 2]
```diff
@@ -4,7 +4,11 @@ import styles from "./Form.module.scss";
...
+  roomNumber: string;
+}
+
+export const Form = (props: Roo) => {
   const {t} = useTranslation();
   const {modalActive, setModalActive} = useModalVisibilityContext();
```
Comment (tritonJS826) [inline gf-webapp/src/logic/bookingPage/form/Form.tsx]: let's name boolean variables with the prefix "is". What do u think?

File: gf-webapp/src/logic/bookingPage/form/Form.tsx [hunk 3]
```diff
@@ -42,6 +46,10 @@ export const Form = () => {
     setToSend({...toSend, [e.target.name]: e.target.value});
   };
 
+  const handleChangeRoom = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setToSend({...toSend, [e.target.name]: props.roomNumber});
```
Comment (tritonJS826) [inline gf-webapp/src/logic/bookingPage/form/Form.tsx:50]: don't use third-party libraries for this task (sending mail). What happens if your services will not work but the customer will get an email? U should handle changes ob your own backend

File: gf-webapp/src/logic/bookingPage/form/Form.tsx [hunk 4]
```diff
@@ -151,6 +148,7 @@ export const Form = () => {
                 name="adults_number"
```
Comment (tritonJS826) [inline gf-webapp/src/logic/bookingPage/form/Form.tsx]: number vs amount. Is it different?)

File: gf-webapp/src/logic/bookingPage/hotelPlan/HotelPLan.tsx [hunk 5]
```diff
@@ -1,27 +0,0 @@
...
-import styles from "./HotelPlan.module.scss";
-import hotelPlan from "../../../resources/hotelPlan/hotelPlan.jpg";
-
-export const HotelPlan = () => {
-  const {t} = useTranslation();
-  return (
-    <div className={styles.wrapper}>
-      <div className={styles.single}>
-        <button className={`${styles.block} ${styles.block1}`}>
-          {t("singleRoom")}
```
Comment (tritonJS826) [inline gf-webapp/src/logic/bookingPage/hotelPlan/HotelPLan.tsx:12]: strings as keys is not type safe

**Review by tritonJS826 — CHANGES_REQUESTED** [review]

File: gf-webapp/src/logic/bookingPage/form/Form.tsx (hunk 2 above)
Comment (tritonJS826) [reply gf-webapp/src/logic/bookingPage/form/Form.tsx]: const {ismodalActive, setIsModalActive } = ..

File: gf-webapp/src/logic/bookingPage/hotelPlan/HotelPLan.tsx (hunk 5 above)
Comment (Ekaterina1994) [reply gf-webapp/src/logic/bookingPage/hotelPlan/HotelPLan.tsx:12]: This file was deleted. Now we use other component for hotel plan

File: gf-webapp/src/logic/bookingPage/form/Form.tsx (hunk 4 above)
Comment (Ekaterina1994) [reply gf-webapp/src/logic/bookingPage/form/Form.tsx]: Change to amount

File: gf-webapp/src/logic/bookingPage/form/Form.tsx (hunk 3 above)
Comment (Ekaterina1994) [reply gf-webapp/src/logic/bookingPage/form/Form.tsx:50]: Created a new issue with this task

File: gf-webapp/src/logic/bookingPage/form/Form.tsx (hunk 2 above)
Comment (Ekaterina1994) [reply gf-webapp/src/logic/bookingPage/form/Form.tsx]: Done

File: gf-webapp/src/logic/bookingPage/form/Form.tsx (hunk 1 above)
Comment (Ekaterina1994) [reply gf-webapp/src/logic/bookingPage/form/Form.tsx]: Done

**Review by scorpigg — APPROVED** [review]

**Pull request merged**